src/test/java/com/google/research/bleth/simulator/AwakenessStrategyTest.java

```diff
@@ -0,0 +1,102 @@
+package com.google.research.bleth.simulator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.runner.RunWith;
+import org.junit.Test;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public abstract class AwakenessStrategyTest {
+    @Test
+    public void observerWithFirstAwakenessTimeZeroStartsAwake() {
+        IAwakenessStrategy awakenessStrategy = createAwakenessStrategy(5, 3, 0);
```

Comment:
Google best practices recommend to document ambiguous literal arguments at the call site (i.e., any argument whose meaning isn't obvious - due to being a named constant, variable, etc.)

So this would become

```
  createAwakenessStrategy(
      /* awakenessCycleDuration= */ 5, 
      /* awakenessDuration= */ 3,
      /* firstAwakenessTime= */ 0);
```

(applies to all other tests as well, obviously)

[Google internal tooling would also report an error if the parameter name does not match the comment, but we don't have that in Github :-( ]